specification/dimensions/usagetype.md

```diff
@@ -0,0 +1,42 @@
+# Usage Type
+
+UsageType is a categorization parameter that defines how a resource or service was consumed during a specified period. The UsageType can influence the price point and also indicate whether purchased commitments are used or unused. This classification offers a nuanced view into the resource or service's consumption patterns and helps in understanding how you're billed, especially when considered alongside attributes like ChargeType and PurchaseType.
+
+The UsageType column MUST be present and MUST NOT be null or empty. This column is of type String and MUST be one of the
+allowed values.
+
+See [Appendix: Usage Type](#usagetype-1) for details about allowed values and governance criteria for this dimension.
+
+## Column ID
+
+UsageType
+
+## Display Name
+
+Usage Type
+
+## Description
+
+Indicates whether the record pertains to an on-demand or recurring charge, or relates to the cost or capacity using a committed discount rate or from unutilized commitments.
+
+## Content Constraints
+
+| Constraint      | Value                                    |
+| :-------------- | :--------------------------------------- |
+| Column required | True                                     |
+| Data type       | String                                   |
+| Allows nulls    | False                                    |
+| Value format    | list-of-values                           |
+
+Allowed values:
+
+| Value      | Description                                                                                                                                                                   |
+|:-----------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| On-Demand         | This usage type describes the consumption of resources or services that are billed at the standard on-demand unit price.                                                |
+| Interruptible     | The interruptible usage type pertains to resources that are provisioned using spot availability. These resources are billed at a fluctuating spot price, which is typically lower than on-demand prices.                                                                                  |
+| Committed         | Committed usage refers to the consumption of resources that benefit from a discounted rate due to long-term commitments, such as reserved instances or savings plans. These resources are billed at this reduced rate. |
+| Unused Commitment | This usage type indicates a scenario where a customer has committed to a specific amount or capacity (hourly, daily, etc.) but hasn't fully utilized it. The line item reflects the capacity or monetary value that went unused.                                                                        |
```

Comment:
Could we not have spaces in the names?

Reply:
+1 on this being a spec-wide decision. Where it would run into issues is things like Service Category values where without spaces, its not going to be very user friendly.

Reply:
One of the advantages of "no space inside names" is that it makes searching for the terms easiers. And it turn these names into "keywords" rather than usual language, reducing the risk of ambiguities.

So it might be more user-friendly for non tech people to have spaces, but it makes life easier in several cases.

I think hashtags have popularized names with no spaces in the "general population".

Reply:
I'm going to resolve this here and propose that we bring this up outside of this column and discuss it holistically. We can then make a sweeping change to apply whatever we decide.